### PR TITLE
Add function to format date and apply across all uses of dates

### PR DESCRIPTION
**Pull request by mjoyce91**
Adds a function to convert dates as returned by the API to a custom format. Existing dates have been updated to use this new function.

### CODE REVIEW

**Comment by mjoyce91** [comment]
`new Date()` accounts for timezone, but the dates we're using aren't timezone-bound. Need to update to ignore timezone.

File: src/utilities.test.js [hunk 1]
```diff
@@ -224,3 +225,32 @@ describe('distanceInWords', () => {
     expect(timeDistanceInWords).toContain(' ago');
   });
 });
+
+describe('formatDate', () => {
+  it('returns a properly formatted date', () => {
+    // how we expect the date from the API
+    const unformattedDate = '2017-01-15';
+    // converted date
+    const formattedDate = formatDate(unformattedDate);
```
Comment (burgwyn) [inline src/utilities.test.js]: You're using addOneDay=true by default

Comment (mjoyce91) [reply src/utilities.test.js]: The tests pass locally that way though. When a date such as "2017-01-01" is provided (like we'd see it in the API), it gets converted to "Sat Dec 31 2016 19:00:00 GMT-0500 (EST)" for us on the East Coast. So assuming we want every one to see the same date, regardless of where they're located, we'd need to ignore the timezone.

Comment (mjoyce91) [reply src/utilities.test.js]: Trying to figure out the best way to do that.

**Comment by mjoyce91** [comment]
Looks like this answer fixed timezone issues - https://github.com/date-fns/date-fns/issues/489#issuecomment-302316888

**Pull request merged**